linux/jdk/redhat/src/packageTest/java/packaging/RedHatFlavoursWithYum.java

```diff
@@ -34,6 +34,9 @@
 public class RedHatFlavoursWithYum implements ArgumentsProvider {
 	@Override
 	public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
+		String arch = System.getProperty("os.arch");
+		boolean ubi7_supported = "x86-64".equalsIgnoreCase(arch) || "amd64".equalsIgnoreCase(arch) || "ppc64le".equalsIgnoreCase(arch) || "s390x".equalsIgnoreCase(arch);
```

Comment:
UBI doesn’t support s390x or ppc64le either

```suggestion
		boolean ubi7_supported = "x86-64".equalsIgnoreCase(arch) || "amd64".equalsIgnoreCase(arch);
```

Reply:
I may be misunderstanding the website, but are these not the same image but for different architectures?

s390x: https://catalog.redhat.com/software/containers/ubi7/ubi/5c3592dcd70cc534b3a37814?container-tabs=technical-information&architecture=s390x

ppc64cle: https://catalog.redhat.com/software/containers/ubi7/ubi/5c3592dcd70cc534b3a37814?container-tabs=technical-information&architecture=ppc64le

Reply:
You're absolutely right! Ignore me :)